### DISCUSSION TITLE
Vanadium vs Bismuth

### DISCUSSION THREAD

**Discussion by ToBinio** [theomenden/TheOmenDen.Vanadium#36] [Q&A] [answered]
quick question.
What's the difference between Vanadium and Bismuth?

as based on their descriptions they seem to be the same.

**Comment by AluTheCrow** [comment, marked as answer]
Primarily due to the way I interface with sodium has changed.

It's not really "just a backport" at the moment. This mod also started off as vanilla mappings from the get-go.

There are a few other differences in the registry setup as well.

However, this should still be able to process namespaces in bismuth, vanadium, optifine, and colormatic.
I will make sure vanadium maintains parity as well.
